code/game/objects/items/weapons/implants/implant_mindshield.dm

```diff
@@ -12,31 +12,22 @@
 				<HR>
 				<b>Implant Details:</b><BR>
 				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
-				<b>Special Features:</b> Will prevent and cure most forms of brainwashing.<BR>
+				<b>Special Features:</b> Will prevent most forms of brainwashing.<BR>
 				<b>Integrity:</b> Implant will last so long as the nanobots are inside the bloodstream."}
 	return dat
 
 
 /obj/item/implant/mindshield/implant(mob/target)
 	if(..())
-		if(target.mind in SSticker.mode.head_revolutionaries || is_shadow_or_thrall(target))
-			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
-			removed(target, 1)
-			qdel(src)
-			return -1
-		if(target.mind in SSticker.mode.revolutionaries)
```

Comment:
This is a big deal. Rev is rarely played, but mindshields are the only reliable way of countering revs. Removing this essentially makes rev unwinnable for the crew. Well.... even more unwinnable than it already is.

Reply:
I added back Loyalty Implants that can be part of ERT kit, or spawned for the ert since rev is admin only , can also replace the mindshield implants of centcom personnel for rp reasons.